pkg/runtime/serializer/versioning/versioning.go

```diff
@@ -132,6 +168,7 @@ func (c *codec) Decode(data []byte, defaultGVK *unversioned.GroupVersionKind, in
 		targetGV.Group = group
 		targetGV.Version = runtime.APIVersionInternal
 	} else {
+		fmt.Printf("looking for %v in %#v\n", group, c.decodeVersion)
```

Comment:
This is probably from my change, but we should not log this debugging message here. Please remove?